ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -374,7 +366,7 @@ def import_stage(self,harvest_object):
             # Find any extras whose values are not strings and try to convert
             # them to strings, as non-string extras are not allowed anymore in
             # CKAN 2.0.
-            for key in package_dict['extras'].keys():
+            for key in package_dict.get('extras', {}).keys():
```

Comment:
You mean check the value value ;) 

Is the intention that default_extras are always added, but only override existing values if override_extras=True?